### ISSUE TITLE
loss.backward() 报错

### ISSUE THREAD

**Issue by JN-Yang** [huangxingKA/My_ISNet#1]
这个代码一直会在loss.backward()这里报错，并且我发现输出的loss值是空值。您有遇到这个问题么？
